requirements.txt

```diff
@@ -2,7 +2,7 @@
 
 numpy>=1.17.2
 torch>=1.8.*
-tqdm>=4.41.0
+tqdm>=4.57.0
```

Comment:
The fix requires updating tqdm?

Reply:
yes. it was failing on lower versions: https://github.com/PyTorchLightning/pytorch-lightning/runs/6262322524?check_suite_focus=true